GameDataParser/Parsers/MapEntityParser.cs

```diff
@@ -17,6 +17,49 @@ public static class MapEntityParser
 
         public static List<MapEntityMetadata> Parse(MemoryMappedFile m2dFile, IEnumerable<PackFileEntry> entries)
         {
+            // Iterate over preset objects to later reference while iterating over exported maps
+            Dictionary<string, string> mapObjects = new Dictionary<string, string>();
+            foreach (PackFileEntry entry in entries)
+            {
+                if (!entry.Name.StartsWith("flat/presets/presets object/"))
+                {
+                    continue;
+                }
+
+                string objStr = entry.Name.ToLower();
+
+                if (string.IsNullOrEmpty(objStr))
+                {
+                    continue;
+                }
+                if (mapObjects.ContainsKey(objStr))
```

Comment:
Can there be an empty line above this

Reply:
I think it makes more sense without a newline since they are part of the same check block.